tests/test_updater_fetch_target.py

```diff
@@ -84,12 +87,39 @@ def _init_updater(self) -> Updater:
             content=b"dir target content",
             encoded_path="a%2Fb%2Fc%2Ftargetpath",
         ),
+        "delegated role standard case": TestTarget(
+            path="projectA/file_a.txt",
+            content=b"file_a content",
+            encoded_path="projectA%2Ffile_a.txt",
+            rolename="A",
+            delegated_role=DelegatedRole(
+                "A", [], 1, False, ["projectA/**"], None
```

Comment:
`"projectA/**"`: the double asterisk format (as it's used in many places) is not supported in python-tuf so this looks misleading.